### PR TITLE
Update user guide for v1.2

### LINKED ISSUE #61: Update User Guide for profiles in v1.2
- [x] Add find profile command `profile -f NAME`
- [x] Add edit profile command `profile -e [n/NAME] [p/PHONE] [e/EMAIL]`
- [x] Update view profile command from `profile -v [INDEX]` to `profile -v`
- [x] Update add profile to from `profile -a n/NAME p/PHONE_NUMBER e/EMAIL` to `profile -a n/NAME p/PHONE_NUMBER e/EMAIL [t/TAGS]...`

### PR DESCRIPTION
**Pull request by lesterong**
### View deployed changes [here](https://lesterong.github.io/tp/UserGuide.html).

Closes #61. 

<hr>

### Changelog

**Profiles**
- Add find profile command `profile -f KEYWORDS [MORE_KEYWORDS]`
- Add edit profile command `profile -e [n/NAME] [p/PHONE] [e/EMAIL]`
- Update view profile command from `profile -v [INDEX]` to `profile -v`
- Update add profile to from `profile -a n/NAME p/PHONE_NUMBER e/EMAIL` to `profile -a n/NAME p/PHONE_NUMBER e/EMAIL [t/TAGS]...`

**Events**
- Add find event command `event -f KEYWORDS [MORE_KEYWORDS]`
- Update view event command from `event -v [INDEX]` to `event -v`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -94,13 +130,13 @@ Deletes a specified profile from NUScheduler.
...
 * Deletes the profile at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+* The index refers to the index number shown in the displayed profile list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Example:
 * `profile -d 1` deletes the first profile listed.
 
-### Adding event: `event -a`
+### Adding a event: `event -a`
```
Comment (ShawnChew) [inline docs/UserGuide.md:139]:
```suggestion
### Adding an event: `event -a`
```

**Review by ShawnChew — APPROVED** [review]
Just some nits. LGTM.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -73,19 +73,55 @@ Format: `help`
...
-Format: `profile -a n/NAME p/PHONE_NUMBER e/EMAIL`
+Format: `profile -a n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A profile can have any number of tags (including 0)
+</div>
+
+Examples:
+* `profile -a n/John Doe p/98765432 e/johnd@example.com t/friend`
+* `profile -a n/Betsy Crowe e/betsycrowe@example.com p/1234567 t/professor`
```
Comment (ShawnChew) [inline docs/UserGuide.md:84]: Maybe you can include an example where more than one tag is used 

```suggestion
* `profile -a n/Betsy Crowe e/betsycrowe@example.com p/1234567 t/professor t/lecturer`
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -73,19 +73,55 @@ Format: `help`
 
 Adds a profile to NUScheduler.
 
-Format: `profile -a n/NAME p/PHONE_NUMBER e/EMAIL`
+Format: `profile -a n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A profile can have any number of tags (including 0)
```
[the remark this answers is not in the document]
Comment (lesterong) [reply docs/UserGuide.md:79]: I just followed what was in the original AB3 UG, but I think this is a good suggestion.

File: docs/UserGuide.md (hunk 2 above)
Comment (lesterong) [reply docs/UserGuide.md]: Good idea. Great catches over all. Thanks Shawn 👍🏼.

**Pull request merged**